### ISSUE TITLE
Usage with TypeScript

### ISSUE THREAD

**Issue by balazssagi** [NicholasBoll/cypress-pipe#5]
I'm trying to use this great library in a TypeScript project. When compiling, I get the following error:

```bash
/node_modules/cypress-pipe/index.d.ts
(20,33): Namespace 'Cypress' has no exported member 'Timeoutable'.
```

I think it has to do with the way TypeScript merges namespaces [(docs)](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-namespaces).

Do you have any idea how to resolve this?

**Comment by NicholasBoll** [comment]
Thank you for filing an issue! I'll look into that today and get back with an answer (and possibly a patch)

**Comment by NicholasBoll** [comment]
This package's type definitions assume the `Cypress` namespace is already available somehow. Most of the time it is provided by the project's `tsconfig.json` file via `"types": ["cypress"]` or something similar.

#6 explicitly references the Cypress type definitions and should fix your problem. I'll reply back with a version you can try. Otherwise the workaround is to explicitly reference the Cypress types somewhere (`tsconfig.json` or anywhere else)

**Comment by NicholasBoll** [comment]
This issue was automatically closed by the merging of #6. The version is `1.3.2`. Try upgrading to `1.3.2` and see if that fixes your issue.

If not, you'll have to get the `Cypress` namespace into the `global` namespace somehow (either by using `/// <reference types="cypress" />` or adding `"cypress"` into `"types"` in your project's `tsconfig.json` file.
